### ISSUE TITLE
Vectorizer's V.xxxClass() does not work on SVG elements

### ISSUE THREAD

**Issue by mickeyvip** [clientIO/joint#162]
The way `jQuery` implements `addClass()`, `removeClass()`, `toggleClass()`, `hasClass()` is not suitable for `SVG` elements. It must be done with `attr("class", value)` instead.

The `Vectorizer` being describes as *lightweight jQuery for SVG* most likely should do all the `xxClass()` manipulations correctly, but it **does not**. It just delegates those to `jQuery`, so using them on `SVG` elements (as we do if we use `V`) has no effect, or no desired effect.

Thank you.

**Comment by DavidDurman** [comment]
Vectorizer does not delegate to jQuery for these kind of methods. See: https://github.com/clientIO/joint/blob/master/src/vectorizer.js. It uses browser native methods.

**Comment by mickeyvip** [comment]
Thank you for the quick response!

I was debugging and somehow stepped into `jQuery`'s `addClass`.

After playing some more I cannot reproduce it, but I found another problem.

Now I do:
```javascript
// this.element is jQuery wrapped <g> element
var rect = V(this.element).find("rect");
rect.addClass("my-class")
```
and here I get an exception:
```console
TypeError: rect.addClass is not a function
```

Stepping into code I see that `V.prototype.find()` is mutating the `nodes` array:
```javascript
find: function(selector) {
  var nodes = this.node.querySelectorAll(selector);
  // Map DOM elements to `VElement`s.
  for (var i = 0, len = nodes.length; i < len; i++) {
    nodes[i] = V(nodes[i]);
  }
  return nodes;
}
```
The problem is that `nodes` array is not changed. 
It is happening in Chrome v44, Safari 8, Firefox 39 on MAC.

The following change makes it work:
```javascript
find: function(selector) {
  var nodes = this.node.querySelectorAll(selector);
  var vNodes = [];
  // Map DOM elements to `VElement`s.
  for (var i = 0, len = nodes.length; i < len; i++) {
    vNodes[i] = V(nodes[i]);
  }
  return vNodes;
}
```

or even:
```javascript
find: function(selector) {
  var nodes = this.node.querySelectorAll(selector);
  var tempVNode];
  // Map DOM elements to `VElement`s.
  for (var i = 0, len = nodes.length; i < len; i++) {
    tempVNode = V(nodes[i]);
    nodes[i] = tempVNode;
  }
  return nodes;
}
```

Thank you.

**Comment by mickeyvip** [comment]
Updated to `0.94` and the last problem gone, although the code seems to be the same.

Sorry for the trouble and thank you.

**Comment by kumilingus** [comment]
Hi Mickey,
the issue you mentioned did exist (Not sure why updating the jointjs helped in your case though). Fixed in bdf4940647f8f4680dfb846fb09a70a55f497ce4. Thanks for pointing that out!

**Comment by mickeyvip** [comment]
Thank you!
